src/main/java/io/reactivex/Single.java

```diff
@@ -1875,6 +1875,8 @@
     /**
      * Hides the identity of the current Single, including the Disposable that is sent
      * to the downstream via {@code onSubscribe()}.
+     * <p>
+     * <img width="640" height="470" src="https://raw.githubusercontent.com/UMFsimke/RxJava/Images/Images/Hide.png" alt="">
```

Comment:
There should be a different superhero(ine) per reactive type who is relatively recognizable from the stylized head and doesn't require complicated drawings.

Reply:
Is this one suitable?

https://github.com/UMFsimke/RxJava/blob/Images/Images/Hide2.png

Reply:
Yes.

https://raw.github.com/wiki/ReactiveX/RxJava/images/rx-operators/Single.hide.png

Reply:
Thanks, updated